packages/storybook/stories/NonReact/NonReactComposites.stories.mdx

```diff
@@ -1,86 +1,150 @@
+as a set of javascript bundles. This will allow developers to try out the UI Library with just a `<script>` tag and few lines of code.
 
-You can easily find prebuilt bundles from:
+Why you want to try out the composites via CDN experience:
 
-[The latest bundles for chat composite](https://github.com/Azure/communication-ui-library/releases/latest/download/chatComposite.js)
+- Great for early prototyping
+- Ability to jump into a call/chat thread with minimal engineering investment
+- Don't need to complicate your engineering environment with React and Webpack
 
-[The latest bundles for call composite](https://github.com/Azure/communication-ui-library/releases/latest/download/callComposite.js)
+Why you want to use our React offerings for production workloads:
 
-For development and prototyping purpose, import these 2 urls with `<script>` tags will be the fastest way. In production environment, we recommend that serving composite files on your own CDN will help with performance.
+- Ability to apply performance optimizations (e.g tree-shaking)
+- In-depth visual customization and theming
+- Leverage modern engineering tooling with capabilities such as type-checking
 
-## Use Composites in html
+_We suggest if you would like to continue to use this method of using the UI Library to host the bundled script experience in your own CDN (Content Delivery Networks)_
 
-After importing chat/calling composite bundle, you are able to load composites using these 2 code snippets:
+## Using Composites in your index.html
+
+The first step is to add a script tag
+
+### Calling
+
+`<script src="https://github.com/Azure/communication-ui-library/releases/latest/download/callComposite.js"></script>`
+
+### Chat
+
+`<script src="https://github.com/Azure/communication-ui-library/releases/latest/download/chatComposite.js"></script>`
+
+_If you want to use both calling and chat, add both to your website_
+
+You will be able to load load composites using `chatComposite.loadChatComposite` and `callComposite.loadCallComposite`.
+
+We use a pattern where we specify important arguments to launch the experience, followed by the DOM element to apply the UI into, followed by any additional properties.
+
+### Code to load the chat composite
+
+We start with our necessary arguments (displayName, threadId, endpoint, userId, token) followed by the html element that will contain our chat composite, and the
+any additional properties for the composite.
 
 ```tsx
 const chatAdapter = await chatComposite.loadChatComposite(
   //The for composite which allows you control composite externally,
   // This function will be exposed to window.chatComposite once you import composite bundle
   {
-    displayName, // string
-    threadId, // string
-    endpoint, // string
-    userId, // Object { communicationUserId: string }
-    token // string
+    displayName: 'DISPLAY_NAME', // string
+    threadId: 'THREAD_ID', // string
+    endpoint: 'ENDPOINT', // string
+    userId: 'USERID', //
```

Comment:
Did we change this to string?

Reply:
changing this to : { communicationUserId: 'USERID' }, // CommunicationUserIdentifier